website/client/src/components/Routes/PrivateRouteAdmin.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { IS_ADMIN } from '../../constant/constant';
+import { Redirect, Route } from 'react-router-dom';
+
+const PrivateRouteAdmin = ({ component: Component, ...rest }) =>
+    (
+        <Route {...rest} render={
+            (props) =>
+                (
+                    rest.user ? (
+                        rest.user.role === IS_ADMIN ? <Component /> : <Redirect to='/admin/login' />
```

Comment:
You need to pass all props from Router to render Component, as <Component {...props} />, in order to get all router props in the child component.